templates/login.html

```diff
@@ -12,12 +12,12 @@ <h2 class="heading">Zalogowany jako {{ user.first_name }} {{ user.last_name }}</
       <h2 class="heading">Wybierz metodę logowania: </h2>
       <div class="form-signin">
         <div class="form-signin-button">
-          <a href="{% url 'scopedallaccess-login' 'facebook' %}" class="btn" role="button">
+          <a href="{% url "social:begin" "facebook" %}" class="btn" role="button">
             <img alt="Continue with Facebook" title="Continue with Facebook" src="{% static 'images/login_buttons/facebook/continue_with.png' %}" style="padding: 0.25em;">
```

Comment:
Can we do something about these weirdly sized buttons while we're on this as well?

Reply:
I think the size is based on the options offered by providers?

Reply:
I think you are allowed to adapt them to your website, but you have to carefully read through the "what are you alowed to do" documents. Just taking them as-is was a temporary hack to make them compliant because whoever made the previous ones (whether it's one of the old devs or it came from django-allaccess samples) clearly didn't care. I've never seen social login buttons look as bad as they do on our site now :P

Reply:
We can fix the buttons separatelly. This PR is not about looks, it's about migrating to a new library.

Reply:
Sure, just don't forget to add an isssue about it before we merge this